javascript/node/selenium-webdriver/edge.js

```diff
@@ -198,6 +198,10 @@ class Driver extends webdriver.WebDriver {
    * @return {!Driver} A new driver instance.
    */
   static createSession(options, service = getDefaultService()) {
+    if (!service) {
```

Comment:
Would you mind removing the default parameter above? (I doubt I'll ever use defaults again since you still have to protect against callers explicitly passing `null` or `undefined`)